src/App.jsx

```diff
@@ -1,10 +1,22 @@
 import { ChakraProvider, Button, Image } from '@chakra-ui/react';
 import React, { useState } from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import './App.css';
 import DropZone from './components/DropZone/DropZone';
 import uploadImage from './util/furnitureUtils';
 import DonationForm from './components/DonationForm/DonationForm';
 
+import Dashboard from './pages/dashboard/Dashboard';
+import DonateEdit from './pages/dashboard/DonateEdit';
+import Drivers from './pages/dashboard/Drivers';
+import RoutesPage from './pages/dashboard/Routes';
```

Comment:
consider potentially renaming this page to something more descriptive like `DriverRoutes` to avoid naming confusion from `Routes` in `react-router-dom`